docs/source/installation.rst

```diff
@@ -19,40 +19,78 @@ distribution to view Conda's installation instructions.
 Install and Activate the environment
 ************************************
 
-Once `conda` is installed and available on the path, create and activate the environment using:
+For most end users, simply installing the package is sufficient to use ASPIRE.
+The commands in this section should install ASPIRE directly from the ``Python Package Index`` into your activated environment.
+This does not require checking out source code.
+If you are interested in checking out and working with the source code, running tests, or a different flavor of install,
+then skip to the next section now instead.
+
+Once ``conda`` is installed and available on the path, create and activate the environment using:
 
 ::
 
-    cd /path/to/git/clone/folder
-    conda env create -f environment.yml
-    conda activate aspire
+   conda create -n aspire_env python=3.6 pip
+   conda activate aspire_env
+   pip install aspire
 
-Test the package
-****************
+.. note::
+    Installing the package installs ASPIRE to the ``site-packages`` folder of your active environment.
+    This is only desirable if you are not going to be doing any development on ASPIRE,
+    but simply want to run scripts that depend on the ASPIRE package.
 
-Make sure all unit tests run correctly by doing:
+
+Alternative Developer Installations
+************************************
+
+Developers are expected to be able to manage their own code and environments.
+However, for consistency and newcomers, we recommend the following procedure using `conda`.
 
 ::
 
-    cd /path/to/git/clone/folder
-    python setup.py test
+   # Acquire the code.
+   git clone git@github.com:ComputationalCryoEM/ASPIRE-Python.git
```

Comment:
To avoid the instruction of setting ssh key, we can just use a simpler way, "git clone https://github.com/ComputationalCryoEM/ASPIRE-Python -b develop". We should request to clone develop branch for general develop purpose.

Reply:
sure, I don't care how people clone their code... It is concerning we need to write it out... changing